src/app/backend/apihandler.go

```diff
@@ -74,6 +74,15 @@ func CreateHttpApiHandler(client *client.Client) http.Handler {
 			Writes(Logs{}))
 	wsContainer.Add(logsWs)
 
+	eventsWs := new(restful.WebService)
+	eventsWs.Path("/api/events").
+		Produces(restful.MIME_JSON)
+	eventsWs.Route(
+		eventsWs.GET("/{namespace}").
```

Comment:
I started implementation with only namespace as a parameter, but most probably this will have to be updated. I read the Kubernetes API documentation and to get events fot specific pod we will have to use `fieldSelectors` with specific parameters like `involvedObject.name=<pod>` or `involvedObject.uid=<pod_uid>`. I will continue to work on it, but if you want I will do it via only one pull request. More documentation will follow code updates.